physionet-django/events/models.py

```diff
@@ -7,6 +7,7 @@
 from events.enums import EventCategory
 from events import validators
 from project.modelcomponents.fields import SafeHTMLField
+from project.modelcomponents.generic import BaseInvitation
```

Comment:
I don't think this import is needed (also, I'm not sure that having a dependency on the project app is a good idea...perhaps the reason that it was not eventually used?)

Reply:
Yup, Exactly!